server/queries/class/getActivitiesQuery.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AssignmentStudent, Assignment, Class, ClassStudent, Student, User,
+} from '../../models';
+
+const getActivitiesQuery = (classId: string) => AssignmentStudent.findAll(
+  {
+    raw: true,
+    nest: false,
```

Comment:
Why this query?

Reply:
i deleted it cause there is no need for this file anymore